be/src/bench/persistent_index_bench.cpp

```diff
@@ -144,11 +144,13 @@ void PersistentIndexBenchTest::do_bench(benchmark::State& state) {
         ASSERT_CHECK(_index->commit(&_index_meta, &stat));
         uint64_t tail = watch.elapsed_time();
         ASSERT_CHECK(_index->on_commited());
+        /* `_index->get_write_amp_score` interface is removed
```

Comment:
@luohaha please advice if this fix is correct or not.

Reply:
It is ok